pypxe/http.py

```diff
@@ -55,7 +61,11 @@ def handleRequest(self, connection, addr):
         self.logger.debug('  <--BEGIN MESSAGE-->\n\t{request}\n\t<--END MESSAGE-->'.format(request = repr(request)))
         startline = request.split('\r\n')[0].split(' ')
         method = startline[0]
-        target = startline[1]
+        req_file = startline[1]
+
+        # avoid directory traversal: strip all ../ and make it relative
+        target = os.path.normpath(os.sep + self.netbootDirectory + os.sep + req_file).lstrip(os.sep)
```

Comment:
Or maybe for the prefixed `/` an `os.sep.join((self,netbootDirectory, req_file))` would be more suitable.